### ISSUE TITLE
The request was aborted: Could not create SSL/TLS secure channel

### ISSUE THREAD

**Issue by GlennIM** [pexels/PexelsDotNetSDK#6]
When I call this method an exception is thrown...

var result = await pexelsClient.SearchVideosAsync("Nature");

InnerException: {"The request was aborted: Could not create SSL/TLS secure channel."}
Message: "An error occurred while sending the request."

How do I fix this?

**Comment by scgough** [comment]
Hi @GlennIM - are you still experiencing this issue? It could have been a temporary problem as this seems to be working fine now.

**Comment by GlennIM** [comment]
Yes. Still the same error. It's being called in a WinForm c# application in the Load() event method. I'm using Visual Studio Express 2017 and .NET Framework 4.8. Please help,

        private async void Form1_LoadAsync(object sender, EventArgs e)
        {
            var pexelsClient = new PexelsClient("MY API KEY");

            try
            {
                var result = await pexelsClient.SearchVideosAsync("Nature");
            }
            catch(Exception ex)
            {
                MessageBox.Show(ex.Message);
                return;
            }
     }
